.github/workflows/tests.yml

```diff
@@ -114,3 +114,39 @@ jobs:
           docker run --rm -e COMPOSER_ROOT_VERSION -v $(pwd):/app -v $(which composer):/usr/local/bin/composer -v /usr/local/bin/vulcain:/usr/local/bin/vulcain -w /app php:7.4-alpine ./phpunit src/Symfony/Component/HttpClient/Tests/CurlHttpClientTest.php --filter testHttp2Push
           sudo rm -rf .phpunit
           [ -d .phpunit.bak ] && mv .phpunit.bak .phpunit
+
+  nightly:
+    name: PHPUnit on PHP nightly
+    runs-on: Ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "none"
+          ini-values: "memory_limit=-1"
+          php-version: "8.1"
+
+      - name: Configure composer
+        run: |
+          COMPOSER_HOME="$(composer config home)"
+          composer self-update
+          ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
+          echo "COMPOSER_ROOT_VERSION=$(grep -m1 SYMFONY_VERSION .travis.yml | grep -o '[0-9.x]*').x-dev" >> $GITHUB_ENV
+
+      - name: Install dependencies
+        run: |
+          echo "::group::fake PHP version"
+          composer config platform.php 8.0.99
+          echo "::group::composer update"
+          composer update --no-progress --ansi
+          echo "::endgroup::"
+          echo "::group::install phpunit"
+          ./phpunit install
+          echo "::endgroup::"
+
+      - name: Run tests
+        run: find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -not -wholename '*/Bridge/PhpUnit/*' -print0 | xargs -0 -n1 dirname | sort | parallel ./phpunit --colors=always --exclude-group tty,benchmark,intl-dat
```

Comment:
Maybe this is too much, but if we put `./phpunit --colors=always --exclude-group tty,benchmark,intl-dat` in a bash function. We may also add something like this: 

```
_run_tests() {
  echo ::group::$1
  ok=0
  ./phpunit --colors=always --exclude-group tty,benchmark,intl-dat ./$1 2>&1 || ok=1
  echo ::endgroup::
  if [ $ok -ne 0 ]; then
    echo "::error::$1 failed"
  fi
  
  return $ok
}
```

Reply:
Let's try that.

Reply:
Yeah.. But Github Action does not really work ATM.. 

https://www.githubstatus.com/

Reply:
🙁 

All right, let's wait for them to fix it. 🤷🏻‍♂️

Reply:
GHA seem to be back. I've triggered a re-run.

Reply:
Looks like the bash function is not found. What did I miss? 🙈

Reply:
We are missing knowledge how Parallel really works. See here: 

https://stackoverflow.com/questions/23814360/gnu-parallel-and-bash-functions-how-to-run-the-simple-example-from-the-manual

Reply:
That did the trick, thanks! 🙏🏻